src/utils/index.ts

```diff
@@ -13,4 +13,22 @@ export * from './displayTo';
 export * from './keyCodes';
 export * from './text';
 export * from './labels';
-export { default as styled, css, keyframes, injectGlobal, withTheme, themed, reStyled, ThemeProvider } from './styled';
+export {
+  default as styled,
+  css,
+  keyframes,
+  createGlobalStyle,
+  withTheme,
+  ThemeProvider,
+  ThemeConsumer,
+  ThemeContext,
+  isStyledComponent,
+  ServerStyleSheet,
+  StyleSheetManager,
+  themed,
+  /**
+   * @deprecated
+   * use `styled` instead
```

Comment:
Hm could also be added to Pharos UI in the compat layer and removed from Precise UI.